app/controllers/concerns/foreman/controller/session.rb

```diff
@@ -26,6 +26,7 @@ def update_activity_time
   end
 
   def set_activity_time
+    return if session[:sso_method] == 'SSO::Jwt' || session[:sso_method] == 'SSO::OpenidConnect'
```

Comment:
This is because for SSO::OpenIdConnect we will use the token's expiry time as the session's expiry time. This is handled using the `store` method in openid_connect.rb ([here](https://github.com/theforeman/foreman/pull/6549/files#diff-35a42e671c71f91179a8032457cd62b5R40)) and set_current_user method in authentication.rb ([here](https://github.com/theforeman/foreman/pull/6549/files#diff-f41d9ad6a64c4df8265f8794634c27beR82))

My assumption is that SSO::Jwt will also use the token's expiry time. If that's not correct, I can remove SSO::Jwt from here and have it only for SSO::OpenidConnect.

Reply:
Update: I removed the condition for SSO::Jwt